plugin/goto.py

```diff
@@ -36,9 +35,6 @@ class LspGotoCommand(LspTextCommand):
 
     method = ''
 
-    def is_enabled(self, event: Optional[dict] = None, point: Optional[int] = None) -> bool:
-        return super().is_enabled(event, point) and is_at_word(self.view, event, point)
-
```

Comment:
I'm not sure why have you removed the `is_at_word` check. Is it because you thought it was not effective or because you think we should be able to trigger "go to" on non-word selection also?

Reply:
I've removed unrelated changes.